### DISCUSSION TITLE
what nationality are you

### DISCUSSION THREAD

**Discussion by stuffbymax** [damiandudycz/ps3#16] [General]
are you czech?

**Comment by damiandudycz** [comment]
Hi. Im Polish. We like Czech in Poland :)

**Comment by stuffbymax** [comment]
czech like pols too

**Comment by stuffbymax** [comment]
damiandudycz do tou have any non ps3 code related stuff

**Reply by damiandudycz** [reply]
I have some public repositories for swift stuff, iOS programming, and ESP32 on macOS.
